TASVideos/TagHelpers/BootstrapTagHelpers.cs

```diff
@@ -223,7 +223,7 @@ public override async Task ProcessAsync(TagHelperContext context, TagHelperOutpu
 		<div class='modal-content'>
 			<div class='modal-header'>
 				<h5 class='modal-title text-danger'>Delete Warning!</h5>
-				<button type='button' class='btn-close float-end' data-bs-dismiss='modal'><span aria-hidden='true'>&times;</span></button>
+				<button type='button' class='btn-close' data-bs-dismiss='modal'></button>
```

Comment:
What about simply replacing it with:
`<i id="forum-header-dismiss" class="float-end fa fa-close" style="cursor: pointer"></i>`
Wouldn't be the same? I didn't make a PR because I couldn't test it myself, so I wonder it that would work.
![image](https://user-images.githubusercontent.com/36802254/149504481-cc49e03d-6bdc-4708-8edd-51a62e941588.png) also, I couldn't set the margin on the left.

Reply:
No.
This is a bootstrap function that dismisses the modal that has been opened. Read this: https://getbootstrap.com/docs/5.1/components/modal/

The forum header dismiss sets a localstorage value so that future visits to the site don't show the header.